src/components/Nav/Content.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Redirect, Route } from 'react-router-dom';
+import SwitchErrorBoundary from '../SwitchErrorBoundary/SwitchErrorBoundary';
+import { routes, pathRoutes } from '../../routes';
+import PfContainerNavVertical from '../Pf/PfContainerNavVertical';
+
+class Content extends React.Component {
```

Comment:
`Content` is a really general term. Everything is Content.
Also, it looks like this `React.Component` could be a [ReactPureComponent](https://reactjs.org/docs/react-api.html#reactpurecomponent) and would save some memory and performance.

Reply:
mmm it's not working with PureComponent

Reply:
it seems there are problems in Purecomponent with routes

Reply:
I don't think is really a `PureComponent`
> If your React component’s render() function renders the same result given the same props and state, you can use React.PureComponent for a performance boost in some cases.

But this one renders different depending the url